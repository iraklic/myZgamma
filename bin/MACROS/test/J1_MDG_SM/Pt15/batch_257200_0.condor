000 (257200.000.000) 10/28 11:42:10 Job submitted from host: <131.225.191.162:59609>
...
001 (257200.000.000) 10/28 11:42:42 Job executing on host: <131.225.191.150:33930>
...
005 (257200.000.000) 10/28 11:42:46 Job terminated.
	(1) Normal termination (return value 1)
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	6427  -  Run Bytes Sent By Job
	37404016  -  Run Bytes Received By Job
	6427  -  Total Bytes Sent By Job
	37404016  -  Total Bytes Received By Job
...
