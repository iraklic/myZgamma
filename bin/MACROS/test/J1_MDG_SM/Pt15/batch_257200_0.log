 PARAMETER DEFINITIONS:
    NO.   NAME         VALUE      STEP SIZE      LIMITS
     1 X0           1.00000e+00  1.00000e-01     no limits
     2 X1           1.00000e+00  1.00000e-01     no limits
     3 X2           1.00000e+00  1.00000e-01     no limits
     4 X3           1.00000e+00  1.00000e-01     no limits
     5 X4           1.00000e+00  1.00000e-01     no limits
     6 X5           1.00000e+00  1.00000e-01     no limits
     7 X6           1.00000e+00  1.00000e-01     no limits
     8 X7           1.00000e+00  1.00000e-01     no limits
     9 X8           1.00000e+00  1.00000e-01     no limits
    10 X9           1.00000e+00  1.00000e-01     no limits
    11 X10          1.00000e+00  1.00000e-01     no limits
    12 X11          1.00000e+00  1.00000e-01     no limits
    13 X12          1.00000e+00  1.00000e-01     no limits
    14 X13          1.00000e+00  1.00000e-01     no limits
    15 X14          1.00000e+00  1.00000e-01     no limits
    16 X15          1.00000e+00  1.00000e-01     no limits
    17 X16          1.00000e+00  1.00000e-01     no limits
    18 X17          1.00000e+00  1.00000e-01     no limits
    19 X18          1.00000e+00  1.00000e-01     no limits
    20 X19          1.00000e+00  1.00000e-01     no limits
 **********
 **    1 **FIX           1
 **********
 **********
 **    2 **FIX           2
 **********
 **********
 **    3 **FIX           3
 **********
 **********
 **    4 **FIX           4
 **********
 **********
 **    5 **FIX           5
 **********
 **********
 **    6 **FIX           6
 **********
 **********
 **    7 **FIX           7
 **********
 **********
 **    8 **FIX           8
 **********
 **********
 **    9 **FIX           9
 **********
 **********
 **   10 **FIX          10
 **********
 **********
 **   11 **FIX          11
 **********
 **********
 **   12 **FIX          12
 **********
 **********
 **   13 **FIX          13
 **********
 **********
 **   14 **FIX          14
 **********
 **********
 **   15 **FIX          15
 **********
 **********
 **   16 **FIX          16
 **********
 **********
 **   17 **FIX          17
 **********
 **********
 **   18 **FIX          18
 **********
 **********
 **   19 **FIX          19
 **********
 **********
 **   20 **FIX          20
 **********
 **********
 **   21 **RELEASE           1
 **********
                   PARAMETER    1  X0 RESTORED TO VARIABLE.
 **********
 **   22 **MIGRAD 
 **********
 FIRST CALL TO USER FUNCTION AT NEW START POINT, WITH IFLAG=4.
Error in <TFile::TFile>: file finalFile.root does not exist

 *** Break *** segmentation violation



===========================================================
There was a crash.
This is the entire stack trace of all threads:
===========================================================
#0  0x0000003cfaa9a115 in waitpid () from /lib64/libc.so.6
#1  0x0000003cfaa3c481 in do_system () from /lib64/libc.so.6
#2  0x00002ab023f652c2 in TUnixSystem::StackTrace() ()
   from /uscmst1/prod/sw/cms/slc5_amd64_gcc434/cms/cmssw/CMSSW_4_2_0/external/slc5_amd64_gcc434/lib/libCore.so
#3  0x00002ab023f65d7c in TUnixSystem::DispatchSignals(ESignals) ()
   from /uscmst1/prod/sw/cms/slc5_amd64_gcc434/cms/cmssw/CMSSW_4_2_0/external/slc5_amd64_gcc434/lib/libCore.so
#4  <signal handler called>
#5  0x00000000004102cc in likelihood(double*, double*) ()
#6  0x00000000004159b6 in myFcn(int&, double*, double&, double*, int) ()
#7  0x00002ab0261cdebb in TMinuit::Eval(int, double*, double&, double*, int)
    ()
   from /uscmst1/prod/sw/cms/slc5_amd64_gcc434/cms/cmssw/CMSSW_4_2_0/external/slc5_amd64_gcc434/lib/libMinuit.so
#8  0x00002ab0261d1f7e in TMinuit::mnamin() ()
   from /uscmst1/prod/sw/cms/slc5_amd64_gcc434/cms/cmssw/CMSSW_4_2_0/external/slc5_amd64_gcc434/lib/libMinuit.so
#9  0x00002ab0261d4b51 in TMinuit::mnmigr() ()
   from /uscmst1/prod/sw/cms/slc5_amd64_gcc434/cms/cmssw/CMSSW_4_2_0/external/slc5_amd64_gcc434/lib/libMinuit.so
#10 0x00002ab0261e05f7 in TMinuit::mnexcm(char const*, double*, int, int&) ()
   from /uscmst1/prod/sw/cms/slc5_amd64_gcc434/cms/cmssw/CMSSW_4_2_0/external/slc5_amd64_gcc434/lib/libMinuit.so
#11 0x00002ab0261de935 in TMinuit::mncomd(char const*, int&) ()
   from /uscmst1/prod/sw/cms/slc5_amd64_gcc434/cms/cmssw/CMSSW_4_2_0/external/slc5_amd64_gcc434/lib/libMinuit.so
#12 0x0000000000416108 in main ()
===========================================================


The lines below might hint at the cause of the crash.
If they do not help you then please submit a bug report at
http://root.cern.ch/bugs. Please post the ENTIRE stack trace
from above as an attachment in addition to anything else
that might help us fixing this issue.
===========================================================
#5  0x00000000004102cc in likelihood(double*, double*) ()
#6  0x00000000004159b6 in myFcn(int&, double*, double&, double*, int) ()
#7  0x00002ab0261cdebb in TMinuit::Eval(int, double*, double&, double*, int)
    ()
   from /uscmst1/prod/sw/cms/slc5_amd64_gcc434/cms/cmssw/CMSSW_4_2_0/external/slc5_amd64_gcc434/lib/libMinuit.so
#8  0x00002ab0261d1f7e in TMinuit::mnamin() ()
   from /uscmst1/prod/sw/cms/slc5_amd64_gcc434/cms/cmssw/CMSSW_4_2_0/external/slc5_amd64_gcc434/lib/libMinuit.so
#9  0x00002ab0261d4b51 in TMinuit::mnmigr() ()
   from /uscmst1/prod/sw/cms/slc5_amd64_gcc434/cms/cmssw/CMSSW_4_2_0/external/slc5_amd64_gcc434/lib/libMinuit.so
#10 0x00002ab0261e05f7 in TMinuit::mnexcm(char const*, double*, int, int&) ()
   from /uscmst1/prod/sw/cms/slc5_amd64_gcc434/cms/cmssw/CMSSW_4_2_0/external/slc5_amd64_gcc434/lib/libMinuit.so
#11 0x00002ab0261de935 in TMinuit::mncomd(char const*, int&) ()
   from /uscmst1/prod/sw/cms/slc5_amd64_gcc434/cms/cmssw/CMSSW_4_2_0/external/slc5_amd64_gcc434/lib/libMinuit.so
#12 0x0000000000416108 in main ()
===========================================================


