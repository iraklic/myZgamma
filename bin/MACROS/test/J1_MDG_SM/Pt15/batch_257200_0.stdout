
CMSSW on Condor

started at Fri Oct 28 11:42:44 CDT 2011

parameter set:
CONDOR_CLUSTER: 257200
CONDOR_PROCESS: 0
CMSSW_DIR: /uscms_data/d2/iraklich/Zgamma/CMSSW_4_2_0
RUN_DIR: /uscms_data/d2/iraklich/Zgamma/CMSSW_3_8_4_1/src/ElectroWeakAnalysis/MultiBosons/bin/MACROS/test/myMinexam
PARAMETER_SET: output_Mu_B_S.txt
NUM_EVENTS_PER_JOB: 

Writing final parameter-set: batch_257200_0.py to RUN_DIR: /uscms_data/d2/iraklich/Zgamma/CMSSW_3_8_4_1/src/ElectroWeakAnalysis/MultiBosons/bin/MACROS/test/myMinexam

run: myMinexam > batch_257200_0.log 2>&1\n

finished at Fri Oct 28 11:42:46 CDT 2011
